FittedSheetsPod/SheetViewController.swift

```diff
@@ -17,10 +17,14 @@ public class SheetViewController: UIViewController {
     /// Automatically grow/move the sheet to accomidate the keyboard. Defaults to false.
     public var autoAdjustToKeyboard = SheetViewController.autoAdjustToKeyboard
     
-    /// Default value for allowPullingPastMaxHeight. Defaults to true.
-    public static var allowPullingPastMaxHeight = true
-    /// Allow pulling past the maximum height and bounce back. Defaults to true.
-    public var allowPullingPastMaxHeight = SheetViewController.allowPullingPastMaxHeight
+	/// Default value for allowPullingPastMaxHeight. Defaults to true.
+	public static var allowPullingPastMaxHeight = true
+	/// Default value for allowPullingPastMinHeight. Defaults to true.
+	public static var allowPullingPastMinHeight = true
```

Comment:
Since this is new functionality, it should default to false (the previous way it worked before)